INDEX.HTML

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html style=" background-color:aqua;
+text-align:center;
+" >
+    <head>
+    
```

Comment:
Welcome, how are you

this example has spaces between the word , but there is no space between the tag. so it is a good practice.

< p >Welcome, how are you < /p>
this practice has spaces between the tags , that is a very bad practice.
it depends about the browser you have that interprete the codes.
this example will always be misinterpreted by a web browser and so lead to an error.

Reply:
[OPTIONAL] 👍🏽 
- I will like to kindly suggest that you go over your comments over again to ensure that all you have written are clear. In this case, I think some words are broken. 👍🏽